docs/framework-docs.md

```diff
@@ -0,0 +1,327 @@
+## Provisioning
+- Packaging (http server -> Lambda function)
+- Static assets: icons only
+  - Serving them out of S3
+  - Validate the contents to be harmless icons??
+
+## Install
+- Manifest -> Installed App
```

Comment:
For HTTP Apps that is true. For AWS Apps we fetch the manifest from S3 and use that.